script/e2e-test.sh

```diff
@@ -118,6 +118,28 @@ if [[ "$code" != 0 ]]; then
   kubectl logs kubeapps-ci-tiller-proxy-test --namespace kubeapps
   echo "LOGS for dashboard tests --------"
   kubectl logs kubeapps-ci-dashboard-test --namespace kubeapps
+
+  exit $code
+fi
+
+# Browser tests
+cd $ROOT_DIR/integration
+kubectl apply -f manifests/executor.yaml
+k8s_wait_for_deployment default integration
+pod=$(kubectl get po -l run=integration -o jsonpath="{.items[0].metadata.name}")
+## Copy latest tests
+kubectl cp ./use-cases ${pod}:/app/
+## Make default user admin
+kubectl create clusterrolebinding default-admin --clusterrole=admin --serviceaccount default:default
```

Comment:
Is this temporary? Or why does the default service account need admin?

Is the plan that in the future different tests can use different roles based on different tokens?

Reply:
Actually, this sounds really dangerous? If I happen to run the browser tests on whatever cluster I have configured, it should *not* be making this change? Let's update to create a specific service account to be used in he ci tests? (and later we can add multiple to test different scenarios of roles?)

Reply:
I don't expect people executing this script in a non testing environment. Note that in the line 54-55 we are already giving admin privileges to the default service account in the `kube-system` namespace, due to an old bug in Minikube. That can be probably be deleted TBH.

In any case, I used the default service account because it was convenient to get the token but I can use something else.

Reply:
Yeah, I wouldn't expect it either, but if they did and didn't realise, it'd be a bad story. I didn't realise that it was already in use in `kube-system`, but yeah, if it's simple to remove and instead create a service account for use, +100, but I'm ok either way. Thanks for clarifying.